pkg/kwokctl/utils/cmd_unix.go

```diff
@@ -0,0 +1,52 @@
+//go:build !windows
```

Comment:
should we name this file to replace `cmd.go`? Or, two files named "cmd_other.go" and "cmd_windows.go".